core/src/main/java/com/linecorp/armeria/client/ClientOptions.java

```diff
@@ -71,31 +73,45 @@
             WRITE_TIMEOUT_MILLIS.newValue(Flags.defaultWriteTimeoutMillis()),
             RESPONSE_TIMEOUT_MILLIS.newValue(Flags.defaultResponseTimeoutMillis()),
             MAX_RESPONSE_LENGTH.newValue(Flags.defaultMaxResponseLength()),
-            DECORATION.newValue(ClientDecoration.NONE),
+            DECORATION.newValue(ClientDecoration.of()),
             HTTP_HEADERS.newValue(HttpHeaders.of())
     };
 
     /**
      * The default {@link ClientOptions}.
+     *
+     * @deprecated Use {@link #of()}.
      */
+    @Deprecated
     public static final ClientOptions DEFAULT = new ClientOptions(DEFAULT_OPTIONS);
 
+    /**
+     * Returns the {@link ClientOptions} with the default options only.
+     */
+    public static ClientOptions of() {
+        return DEFAULT;
+    }
+
     /**
      * Returns the {@link ClientOptions} with the specified {@link ClientOptionValue}s.
      */
     public static ClientOptions of(ClientOptionValue<?>... options) {
         requireNonNull(options, "options");
         if (options.length == 0) {
-            return DEFAULT;
+            return of();
         }
-        return new ClientOptions(DEFAULT, options);
+        return new ClientOptions(of(), options);
```

Comment:
I think we can delegate to the other methods instead of repeating the empty checks `of(of(), options)`

Reply:
That's a good idea.